test/integration/test-node/login-bpmn.ts

```diff
@@ -1,59 +0,0 @@
-/*!
- * @license
- * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import { argv, exit } from 'node:process';
-import { AlfrescoApi } from '@alfresco/js-api';
-import { Command } from 'commander';
```

Comment:
@DenysVuika now we can probably remove this dependency completely from package.json as there is no more usages

Reply:
good catch, removed, thanks